src/execution/mod.rs

```diff
@@ -14,40 +14,50 @@
 //! See [`crate::recovery`] for a description of the recovery
 //! components added to the Timely dataflow.
 
+mod runner;
```

Comment:
We've also been having the Rust-side module structure parallel the Python-side structure. So if we're going to have `bytewax.run` contain `cluster_main` and (the maybe renamed) `_cli_main`, can this Rust module be renamed to `run` from `execution`?

I do think `run_main` should still be continue to be defined next to `cluster_main`, even though we do publicly expose it in `bytewax.testing` just because they are so similar.

Reply:
Ok, I renamed the `execution` module to `run`, and only left there the `run` related things (so moved the worker elsewhere, see other comment)